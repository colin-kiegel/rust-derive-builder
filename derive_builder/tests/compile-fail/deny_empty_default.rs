#[macro_use]
extern crate derive_builder;

// deny `#[builder(default = "")]`, because we don't want to define a meaning (yet)! :-)
#[allow(dead_code)]
#[derive(Builder)]
struct Lorem {
    #[builder(default = "")]
    //~^ ERROR Unknown literal value ``
    ipsum: String,
}

fn main() {}
