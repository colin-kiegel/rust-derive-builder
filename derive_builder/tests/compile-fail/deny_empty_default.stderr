error: unexpected end of input, expected expression
 --> tests/compile-fail/deny_empty_default.rs:8:25
  |
8 |     #[builder(default = "")]
  |                         ^^
